lib/internal/Magento/Framework/Serialize/Serializer/JsonHexTag.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Framework\Serialize\Serializer;
+
+use Magento\Framework\Serialize\SerializerInterface;
+
+/**
+ * Serialize data to JSON, unserialize JSON encoded data
```

Comment:
It would be great to update this inline documentation to indicate difference between this serializer and the other `Json`

Reply:
Improved comment